src/app/layout.tsx

```diff
@@ -21,14 +19,24 @@ export default function RootLayout({
 }: Readonly<{
   children: React.ReactNode;
 }>) {
+  const cookieStore = cookies();
+  const sessionCookie = cookieStore.get('next-auth.session-token');
+  const isLoggedIn = !!sessionCookie;
```

Comment:
```suggestion
const session = await getServerSession(authOptions);
const isLoggedIn = !!session;
```
요렇게 수정해주시면 쿠키 따로 사용 안 해도 될 것 같습니당!!